Jenkinsfile

```diff
@@ -15,10 +15,18 @@ elifePipeline {
 
     elifeMainlineOnly {
         stage 'Deploy on end2end', {
-            lock('elife-xpub--end2end') {
-                builderDeployRevision 'elife-xpub--end2end', commit
-                builderSmokeTests 'elife-xpub--end2end', '/srv/elife-xpub'
-            }
+            elifeSpectrum(
+                deploy: [
+                    stackname: 'elife-xpub--end2end',
+                    revision: commit,
+                    folder: '/srv/elife-xpub',
+                    rollbackStep: {
+                        builderDeployRevision 'elife-xpub--end2end', 'approved'
+                        builderSmokeTests 'elife-xpub--end2end', '/srv/elife-xpub'
+                    }
+                ],
+                marker: 'xpub'
```

Comment:
runs only the end2end tests from `elife-spectrum` marked with `xpub`, as the others do not cover this project